test/lib/createTxMeta.js

```diff
@@ -1,10 +1,10 @@
 import { snapshotFromTxMeta } from '../../app/scripts/controllers/transactions/lib/tx-state-history-helpers'
 
 export default function createTxMeta (partialMeta) {
-  const txMeta = Object.assign({
+  const txMeta = {
     status: 'unapproved',
-    txParams: {},
-  }, partialMeta)
+    txParams: {}, ...partialMeta,
```

Comment:
It's weird to see multi-line object literals with things doubled up on the same line like this 🤔 I would have expected this to be disallowed.

Reply:
I'm not sure why this is allowed. I've fixed it with 4ae9694cf.

Reply:
The same problem is present in _many_ other places, but I'm not gonna block on it. Hopefully prettier will fix it.